test/no-injection-into-hooked/test.k.verify.out.golden

```diff
@@ -1,4 +1 @@
-Error:
-  module 'TEST':
-  axiom declaration:
-    Hooked sorts may not have subsorts.
+kore-parser: ErrorHookedSortWithSubsort {sort = Subsort {subsort = SortActualSort (SortActual {sortActualName = Id {getId = "SortS", idLocation = AstLocationFile (FileLocation {fileName = "./test-kompiled/definition.kore", line = 209, column = 127})}, sortActualSorts = []}), supersort = SortActualSort (SortActual {sortActualName = Id {getId = "SortInt", idLocation = AstLocationFile (FileLocation {fileName = "./test-kompiled/definition.kore", line = 209, column = 136})}, sortActualSorts = []})}, sourceLocation = SourceLocation {location = Location {start = Nothing, end = Nothing}, source = Source {unSource = Nothing}}}
```

Comment:
Does the output look like this because we're not using `koreFail` anymore when throwing the error? What is the reason for that?

Reply:
[This](https://github.com/kframework/kore/pull/2177/commits/6e98f19dfb6acfab388ceb2c2fe0d7152811837d) is some work in progress for pretty printing the error. Does it look like we want to do it?